lib/dice_bag/default_template_file.rb

```diff
@@ -10,12 +10,12 @@ module DiceBag
   class DefaultTemplateFile
     include DiceBagFile
 
-    def initialize(name, location=nil)
+    def initialize(name, location=nil, save_as=nil)
```

Comment:
How about adding spaces around the `=` sign
https://github.com/rubocop-hq/ruby-style-guide#spaces-around-equals

Reply:
Glad you mentioned it❗️ I plan to resolve the style issues in another PR, a lot of the styles in the repo don't seem to fit the coding style guide. 🙊